odoo/service/db.py

```diff
@@ -418,6 +418,12 @@ def list_db_incompatible(databases):
     return incompatible_databases
 
 
+def exp_monodb():
+    dbs = list_dbs(force=True)
```

Comment:
```suggestion
    dbs = list_dbs(force=False)
```
otherwise you bypass the `list_db` configuration parameter

Reply:
Hello,

Since it was fully backward-compatible, I based it on 14.0, but I could open a PR only for master if you prefer.
If you can list databases, you don't need this function. The goal was to bypass the parameter for listing databases in the case where there is only one database, since in that setting, the database management page is often disabled. If it's a too great risk, you can reject this PR.

Another solution would be to change the [authentication API](https://www.odoo.com/documentation/16.0/developer/api/external_api.html#logging-in) to make database optional and choose and return a dict containing the database name (if there is only one) as well as the uid of the user. This way, we could use it for future calls.

Note, that when you enable oauth, you can already see the DB name - in runbot it's also the same as the subdomain.
![image](https://user-images.githubusercontent.com/1943198/214339162-8f71a17d-86d7-4b47-b6ba-d506c8f60c42.png)

Best regards,
Krzysztof